crates/wasm/wasm.nix

```diff
@@ -0,0 +1,33 @@
+{ nixpkgs ? import <nixpkgs> {} }:
+

```

Comment:
what is the purpose of nix here? This module only requires rust so should build reliably I think

Reply:
nix file is left for convenience but is not mandatory for build